src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,14 @@
 
 public class Application {
     public static void main(String[] args) {
```

Comment:
Remember to avoid redundant empty lines after class declaration or method signature as mentioned in the checklist.